bundle/nsx-container-plugin-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1 @@
+../deploy/olm-catalog/0.0.1/nsx-container-plugin-operator.v0.0.1.clusterserviceversion.yaml
```

Comment:
Here is another 0.0.1, not sure if it matters

Reply:
I did not notice this comment. You're correct, this was a broken link.
Updated.